tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js

```diff
@@ -342,34 +344,122 @@ test.describe( 'Set up Ads account', () => {
 				await checkFAQExpandable( page );
 			} );
 		} );
+	} );
 
-		test( 'Set the budget', async () => {
-			budget = '0';
-			await setupBudgetPage.fillBudget( budget );
+	test.describe( 'Set up billing', () => {
+		test.describe( 'Billing status is not approved', () => {
+			test( 'It should say that the billing is not setup', async () => {
+				await page.waitForLoadState( LOAD_STATE.DOM_CONTENT_LOADED );
 
-			await expect(
-				setupBudgetPage.getLaunchPaidCampaignButton()
-			).toBeDisabled();
+				await expect(
+					page.getByRole( 'button', {
+						name: 'Set up billing',
+						exact: true,
+					} )
+				).toBeEnabled();
 
-			budget = '1';
-			await setupBudgetPage.fillBudget( budget );
+				await expect(
+					page.getByText(
+						'You do not have billing information set up in your Google Ads account. Once you have set up billing, you can start running ads.'
+					)
+				).toBeVisible();
+			} );
 
-			await expect(
-				setupBudgetPage.getLaunchPaidCampaignButton()
-			).toBeEnabled();
+			// eslint-disable-next-line jest/expect-expect
+			test( 'should open a popup when clicking set up billing button', async () => {
+				await checkBillingAdsPopup( page );
+			} );
+
+			test( 'should see billing has been set up successfully when billing status API returns approved', async () => {
+				const newPagePromise = page.waitForEvent( 'popup' );
+				await setupBudgetPage.clickSetUpBillingLink();
+				const newPage = await newPagePromise;
+				await newPage.waitForLoadState();
+
+				await setupBudgetPage.fulfillBillingStatusRequest( {
+					status: 'pending',
+				} );
+
+				await newPage.close();
+				await setupBudgetPage.focusBudget();
+				await setupBudgetPage.fulfillBillingStatusRequest( {
+					status: 'approved',
+				} );
+				await setupBudgetPage.awaitForBillingStatusRequest();
+
+				const billingSetupSuccessSection =
+					setupBudgetPage.getBillingSetupSuccessSection();
+				await expect( billingSetupSuccessSection ).toContainText(
+					'Billing method for Google Ads added successfully'
+				);
+			} );
 		} );
+	} );
 
-		test( 'Budget Recommendation', async () => {
-			await expect(
-				page.getByText( 'set a daily budget of 15 USD' )
-			).toBeVisible();
+	test.describe( 'Create Ads with billing data already setup', () => {
```

Comment:
Could it use the same currency within this test group?

https://github.com/woocommerce/google-listings-and-ads/blob/99a5e2c75dedc2d9ab1032b9294177c9a66066b8/tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js#L437

https://github.com/woocommerce/google-listings-and-ads/blob/99a5e2c75dedc2d9ab1032b9294177c9a66066b8/tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js#L453

Reply:
Updated.